apps/api/src/main.ts

```diff
@@ -47,7 +47,12 @@ class CustomLogger implements LoggerService {
 }
 
 async function initializeSentry() {
-  if (!process.env.SENTRY_DSN) {
+  if (
+    !process.env.SENTRY_DSN &&
```

Comment:
`!process.env.SENTRY_DSN || ...` => throw error `Sentry config missing, skipping initialization`

:) you kinda flipped the logic

Alternatively, you can just remove the `!` to the outside.

Reply:
Oopsiee. This is why you don't quickly push. I will be carefull from future! sorry again